R/check_source.R

```diff
@@ -1,5 +1,19 @@
 #' Standardised test for an R source repository
 #'
+#' A convenience function that runs all source related tests in sequence.
+#' The details lists the relevant functions.
```

Comment:
Alternatively to the suggestion, you can leave it as it, but as 'details' is plural, then 'list' has to be without 's'.
```suggestion
#' The details section lists the relevant functions.
```